### ISSUE TITLE
Not compatible with WalletConnect v2

### ISSUE THREAD

**Issue by Popugames** [MoralisWeb3/web3-unity-sdk#70]
How to migrate:
https://docs.walletconnect.com/2.0/advanced/migrating-from-v1.0

**Comment by Eshan204** [comment]
Yes, I'm also facing the same issue while connecting to Moralis while connecting to the Wallet.

**Comment by AnujSahu2024** [comment]
Yes I am also facing the same issue when i trying when connecting to moralis ,when connecting to wallet. Kindly please acknowledge if anyone know how to resolve the issue.

**Comment by sagar164** [comment]
I was working on connecting the smart chain wallet through this SDK earlier it was working fine,
but recently I am getting the same error,

please try to resolve it asap.
